test/arbitrage.test.js

```diff
@@ -91,14 +105,36 @@ test('mapply 2 args with recycling', t => {
   t.true(r.all(r.is_equal(act,exp)))
 })
 
-test.todo('mapply 3 args')
-
+test('mapply 3 args', t => {
+  var act = r.mapply([1,2,3], [1,1,1], [2,2,2], (a, b, c) => a + b + c)
+  var exp = [4,5,6]
+  t.true(r.all(r.is_equal(act, exp)))
+})
 
 // ADD MORE TEST CASES
-test.todo('fold 1')
+test('fold 1', t => {
```

Comment:
This test title needs to be more descriptive. I would say "fold over a vector and compute its sum".

Reply:
Also, this test doesn't test ordering. A better test would use cumulative sum.